core/src/main/AndroidManifest.xml

```diff
@@ -55,4 +69,4 @@
         </intent>
     </queries>
 
-</manifest>
+</manifest>
```

Comment:
Please don't change the line ending of the file

Reply:
@ByteHamster I guess this might have happened because I pushed changes from the command line instead of from within Android Studio, maybe? I just pushed the new changes from Android Studio — could I ask you to confirm that this is resolved?

Reply:
It's probably related to auto-formatting (just like the change above). You can just revert the two auto-formatted changes in Android Studio (or any other editor) manually. In this case, add a line break to the end of the file. Android Studio displays the very last line break, while some other editors don't. Technically, a file needs to end with a line break.